docs/api/blueprint/attendees.apib

```diff
@@ -262,6 +262,394 @@ Get a list of attendees of an order.
           }
         }
 
+## List Attendees under an event [/v1/events/{event_identifier}/attendees]
```

Comment:
Use event ID please. Nevertheless, this test will fail

Reply:
ok, it was working with postman, removing this